modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/ButtonBehavior.java

```diff
@@ -233,4 +234,16 @@ protected void mouseExited(MouseEvent e) {
             getNode().disarm();
         }
     }
+
+    /**
+     * Invoked when the the Button is dragged. If the Button had been armed
+     * by a touch event or a mouse press and the mouse is still pressed,
+     * then this will cause the button to be rearmed. This allows not to fire
```

Comment:
I believe you want to say that this causes the button to be disarmed, and it needs to be rearmed again before it can fire?

Reply:
Probably he means that if the button had been armed, after dragging it would be still armed, and its action would be fired, and by using this method that is prevented.